docs/overview.md

```diff
@@ -1,70 +1,67 @@
-page_title: Docker Registry Service 2.0
-page_description: Introduces the docker registry service
-page_keywords: registry, service, images, repository
+page_title: Docker Registry 2.0
+page_description: Introduces the Docker Registry
+page_keywords: registry, images, repository
 
-# Docker Registry Service 2.0
+# Docker Registry 2.0
 
-The Docker Registry Service stores and distributes Docker images. The majority
-of Docker users pull images from Docker's own public registry instance.
-Installing Docker gives users this ability. Users with a Docker Hub account can
-also push images to this registry.
+Docker Registry stores and distributes images centrally. It's where you push images to and pull them from; Docker Registry gives team members the ability to share images and deploy them to testing, staging and production environments.
 
-A subset of Docker users may wish to deploy a Docker Registry Service of their own. For example, users with their own software products and may want to maintain an image store for private, company use. Some companies also maintain a  registry instance for release of their software images to the public. 
+Docker provides a hosted registry as part of [Docker Hub](https://hub.docker.com). Docker Hub is a cloud service for managing your images, featuring organization accounts, automated builds, and much, much more.
```

Comment:
Docker Hub is a cloud service that securely manages your images. It features organization accounts, automated builds, and much, much more.